aptos-move/e2e-move-tests/src/tests/transaction_context.data/sources/transaction_context_test.move

```diff
@@ -0,0 +1,28 @@
+module 0x1::transaction_context_test {
+    use std::vector;
+    use aptos_framework::transaction_context::generate_unique_address;
+    use std::features;
+
+    /// When checking the value of aggregator fails.
+    const ENOT_UNIQUE: u64 = 20;
+
+    public entry fun create_many_auids(_account: &signer, count: u64) {
+        if (features::auids_enabled()) {
+            let auids: vector<address> = vector<address>[];
```

Comment:
nit: This can just be let auids = vector[];
The type can be inferred

Reply:
Tried it just now. Got this error. So, switching back. 
<img width="938" alt="Screenshot 2023-06-26 at 10 59 41 AM" src="https://github.com/aptos-labs/aptos-core/assets/7092734/7bc2e1bc-ea0d-4a08-97fa-929bbe991619">